analysis/topEFT/datacard_maker.py

```diff
@@ -641,6 +641,10 @@ def addYields(p, name, h_sm, allyields, iproc, signalcount, bkgcount, d_sigs, d_
                        processSyst(name, channel, systMap, d_hists, fout, cat)
                     else:
                        processSyst(proc+'_sm', channel, systMap, d_hists, fout, cat)
+                if p in self.signal:
+                    for bin in range(1, h_sm.GetNbinsX()+1):
+                        h_sm.SetBinError(bin,0)
```

Comment:
I guess the argument here is that background-only it's better than nothing :) 
in any case, it seems to me that we only have problems with low statistics in background samples, which makes sense, because for those we either rely on data or mc that is not necessarily customized to our phase space